proposals/0195-minimum-main-and-secondary-font-size.md

```diff
@@ -0,0 +1,90 @@
+# Minimum Main and Secondary Font Size
+* Proposal: [SDL-0195](https://github.com/MichaelCrimando/sdl_evolution/blob/master/proposals/0195-minimum-main-and-secondary-font-size.md)

```

Comment:
Please change this to `SDL-NNNN`, proposals are only given a number once they have been accepted into review. Also rename the file with the same prefix.